Source/Particles/Pusher/CopyParticleAttribs.H

```diff
@@ -41,30 +39,34 @@ struct CopyParticleAttribs
     /** \brief Construct a new functor
      *
      * \param a_pti iterator to the tile containing the macroparticles
-     * \param tmp_particle_data holder for the temporary particle data
      * \param a_offset offset to apply when reading / writing particle data
      *        This is needed because when we use field gather buffers we don't
      *        always start at the particle with index 0.
      */
-    CopyParticleAttribs (const WarpXParIter& a_pti, TmpParticles& tmp_particle_data,
-                         long a_offset = 0) noexcept
+    CopyParticleAttribs (
+        const WarpXParticleContainer& /*pc*/,
+        WarpXParIter& a_pti,
+        long a_offset = 0) noexcept
     {
-        if (tmp_particle_data.empty()) { return; }
+        // FIXME
+        //if (tmp_particle_data.empty()) { return; }
```

Comment:
@atmyers 

How do you think we should handle this `if` condition that is currently commented out?